### ISSUE TITLE
New server created

### ISSUE THREAD

**Issue by WarDrake** [cjmurph/PmsService#81]
I was already running a configured plex server
When installing PmsService and using my credentials for the service runner, a new server is created

how can I continue to use my current server with PmsService?
already checked there is no new Plex Media Server appdata I'm the only user on the machine, and if I start plex as an app, it picks up it's normal server data.

**Comment by cjmurph** [comment]
Sorry, I'm a bit confused, could you try explaining the issue more clearly?

**Comment by WarDrake** [comment]
Sure, 

When I start PMS from the app, my server "Scion" comes online, with it's libraries and data
When I start PMS from the service, a new server "Nihilus" is started, with blank data, there is no new data folder, anywhere for this server.

but everytime I start the PMS service, it's this server that comes online, not the one I had configured originally.

**Comment by TechieGuy12** [comment]
Is the service running under the same user account you use to start Plex manually?

**Comment by WarDrake** [comment]
Yeah, It's the only account on the computer and I checked the "Default" and "Guest" profiles for a plex data folder, and it's not there either

**Comment by TechieGuy12** [comment]
By default, services run under a service account and not a user account. You would need to manually change the log on account of the service to the user that you use to run Plex manually.

**Comment by cjmurph** [comment]
during install the user should be configured. have you double checked that the service is running as that user?
Have you set a custom location for the library (not the media location, but the plex library files)?
